### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by nkubala**
in theory this was a good idea, but in practice it's causing wrong reviewers to be auto-added for any change that touches files in any of these subdirectories. let's just remove the github owners rules, and tag reviewers as needed for any domain-specific changes.

### CODE REVIEW

**Review by gsquared94 — APPROVED** [review]

**Comment by tejal29** [comment]
The JIB codeowners also helped the jib team to merge their PR without a review from skaffold-team right?

**Comment by tejal29** [comment]
> in theory this was a good idea, but in practice it's causing wrong reviewers to be auto-added for any change that touches files in any of these subdirectories. let's just remove the github owners rules, and tag reviewers as needed for any domain-specific changes.

you can also disable auto-code review [here] (https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team#:~:text=Disabling%20code%20review%20assignment,-In%20the%20top&text=name%2C%20click%20Teams.-,On%20the%20Teams%20tab%2C%20click%20the%20name%20of%20the%20team,Click%20Save%20changes.) instead.

I remember turning this on last year.

**Comment by tejal29** [comment]
I just did this for skaffold-team. If this does not take into effect, we can remove owners. 

image

**Review by tejal29 — CHANGES_REQUESTED** [review]
Do not merge.

**Comment by MarlonGamez** [comment]
@tejal29 any update on this?

**Pull request closed without merging**
